src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -335,9 +335,10 @@ private void extractPageMetadata(final String... aRow) throws CsvParsingExceptio
      * @param aPath A path to a CSV file
      * @return The row
      * @throws CsvParsingException If the row represents a v2 canvas and contains any A/V metadata
+     * @throws MimeTypeParseException
      */
-    @SuppressWarnings({ "unchecked", "PMD.PreserveStackTrace", "PMD.TooFewBranchesForASwitchStatement" })
-    private String[] checkApiCompatibility(final String[] aRow, final Path aPath, final String aIiifVersion)
+    @SuppressWarnings("unchecked")
+    private String[] checkIiifPresApiCompatibility(final String[] aRow, final Path aPath, final String aIiifVersion)
```

Comment:
this

Reply:
Don't know how those slipped through, but they've been fixed.